src/test/resources/logging.properties

```diff
@@ -0,0 +1,10 @@
+handlers = com.logentries.jul.LogentriesHandler
+
+.handlers = com.logentries.jul.LogentriesHandler
+
+com.logentries.jul.LogentriesHandler.token = f2ff093a-54c7-4305-9449-1dccda748347
```

Comment:
As long as it isn't a real token :) You could end up with a lot of unwanted logs.